src/convenience.jl

```diff
@@ -1,7 +1,8 @@
-# This file defines and exports some convenience definition of entropies
-# for commonly used names in the literature. They aren't actually new entropies
-# as discussed extensively in the documentation.
+# This file defines and exports some convenience definition of quantities
+# for commonly used names in the literature. Solely to bridge the
+# gap between calling things that aren't entropies "entropies".
```

Comment:
The last sentence here is incomplete: you need the gap *between* A and B, but B is missing in this sentence. I also don't understand what "calling things that aren't entropies "entropies""" means.